src/components/staking/LiValidator.vue

```diff
@@ -94,10 +94,30 @@ export default {
       type: String,
       /* istanbul ignore next */
       default: () => "returns"
+    },
+    isMultiDenomReward: {
+      type: Boolean,
+      default: false
+    },
+    stakingDenom: {
+      type: String,
+      default: ""
     }
   },
   methods: {
-    percent
+    percent,
+    shortDecimals,
+    selectMostRelevantReward(rewards) {
+      rewards.sort((a, b) => b.amount - a.amount)
+      // we return the reward with the highest amount
```

Comment:
we should choose the staking reward, not the reward with the highest amount.

the "highest amount" has no meaning because we don't know the underlying value of the tokens.

we want to display the staking rewards next to the number of tokens staked.

Reply:
Yes, I already changed it